src/injected/ui.cpp

```diff
@@ -2799,33 +2799,63 @@ PresentPtr &vtable_find(T *obj, int index) {
     return *reinterpret_cast<PresentPtr *>(&ptr[0][index]);
 }
 
-void init_script() {
+void spawn_door_lua(float x, float y, uint8_t l, uint8_t w, uint8_t f, uint8_t t)
+{
+    spawn_door(x, y, l, w, 1, t + 1);
+}
+
+void init_script()
+{
     g_state = (struct StateMemory *)get_state_ptr();
     g_state_addr = reinterpret_cast<uintptr_t>(g_state);
-
-    lua.open_libraries(sol::lib::math);
+    lua.open_libraries(sol::lib::base, sol::lib::math);

```

Comment:
I don't see anything in the commit or comments on why `base` is being added here and I'm having a hard time finding a comprehensive doc on what functions it includes.

Reply:
I believe http://www.lua.org/manual/5.3/manual.html#6.1 covers the scope of what gets pulled in for base. I'm not sure we want to make all of that available just to provide print personally

Reply:
removed base from being included here